.gitignore

```diff
@@ -5,3 +5,9 @@
 vcs/*.smt2
 
 Strata.code-workspace
+
+# For run_cbmc.sh
```

Comment:
Since the files generated by `run_cbmc.sh` aren't the final results of the tool, I'd rather have it delete them when done than have the temporary files listed here. It'd always be possible to comment out a final `rm` line during debugging, when you want to keep the files around.